### PR TITLE
fix massivaction license

### PR DESCRIPTION
**Pull request by Rom1-B**
Missing massive actions to link a license to an item (computer, monitor, network equipment, ...) and vice versa

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

### CODE REVIEW

**Comment by trasher** [comment]
Please fix CS and tests ;)

File: tests/bootstrap.php [hunk 1]
```diff
@@ -151,6 +151,14 @@ function loadDataset() {
             'name'           => '_test_computermodel_2',
             'product_number' => 'CMP_567AEC68',
          ],
+      ], 'Monitor' => [
```
Comment (trasher) [inline tests/bootstrap.php:154]: If you change the tests dataset, you also have to increase its version

Comment (trasher) [reply tests/bootstrap.php:154]: (`_version` at the top of the array)

**Review by trasher — APPROVED** [review]

File: inc/item_softwarelicense.class.php [hunk 2]
```diff
@@ -204,6 +235,27 @@ static function processMassiveActionsForOneItemtype(MassiveAction $ma, CommonDBT
                }
             }
             return;
+
+         case 'add_item':
+            $item_licence = New Item_SoftwareLicense();
```
Comment (cedric-anne) [inline inc/item_softwarelicense.class.php:240]:
```suggestion
            $item_licence = new Item_SoftwareLicense();
```

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**